getting-started/enable-debug-logging.md

```diff
@@ -267,13 +265,18 @@ ACPCore.lifecyclePause();
 {% tab title="Flutter" %}
 ## Flutter
 
-Note: It is required to implement Lifecycle in native [Android and iOS code](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle).
+{% hint style="info" %}
+You need to implement Lifecycle in native Android and iOS code. For more information on implementing, please read the [Lifecycle documentation](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle).
+{% endhint %}
+
 {% endtab %}
 
 {% tab title="Cordova" %}
 ## Cordova
 
-> Note: We recommend implementing Lifecycle in native [Android and iOS code](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle).
+{% hint style="info" %}
+You need to implement Lifecycle in native Android and iOS code. For more information on implementing, please read the [Lifecycle documentation](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle).
```

Comment:
the new link is https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/lifecycle